gui/src/bitcoind.rs

```diff
@@ -114,6 +116,7 @@ impl Bitcoind {
             .map_err(|e| StartInternalBitcoindError::BitcoinDError(e.to_string()))?;
 
         Ok(Self {
+            stdout: process.stdout.take().map(|s| Arc::new(Mutex::new(s))),
```

Comment:
It seems `stderr` would be what we'd be looking for to debug. If we are going to only log stdout maybe we could do a trick like piping stderr to stdout. Or maybe it's fine to just read stderr when we actually detect an error as we do currently.

Reply:
Ignore this, i hadn't seen you actually displayed them on the screen.